src/lib/alligatorUtils.ts

```diff
@@ -122,8 +122,8 @@ export async function getTotalVotableAllowance({
 
   const totalAllowance =
     (allowances.reduce((a, b) => a + b, 0n) *
-      BigInt((1 - subdelegatedShare) * 100000)) /
-      100000n -
+      BigInt(Math.trunc((1 - subdelegatedShare) * 100000))) /
```

Comment:
I don't understand how is this different from the original code?

Reply:
Math.trunc removes the decimals so converts 24931.000000000004 into 24931 so there is no error when converting into a bigint
<img width="936" alt="math trunc-error" src="https://github.com/voteagora/agora-next/assets/29060919/2af28d5d-3775-4ff1-8ccd-ce8134b82640">